avocado/core/xunit.py

```diff
@@ -145,6 +146,26 @@ def add_error(self, state):
                   'reason': self._escape_attr(str(state['fail_reason']))}
         self.testcases.append(tc.format(**values))
 
+    def add_interrupted(self, state):
+        """
+        Add a testcase node of kind interrupted.
+
+        :param state: result of :class:`avocado.core.test.Test.get_state`.
+        :type state: dict
+        """
+        tc = '''\t<testcase classname={class} name={name} time="{time}">
+\t\t<error type={type} message={reason}><![CDATA[{traceback}]]></error>
+\t\t<system-out><![CDATA[{systemout}]]></system-out>
+\t</testcase>'''
```

Comment:
Correct me if I'm wrong, but there is no such type like `interrupted` in XUNIT, so this function is identical to `add_error`. IMO instead of creating yet another duplicate it'd be better to map `INTERRUPTED` state to `add_error`